src/material-experimental/popover-edit/popover-edit-module.ts

```diff
@@ -21,25 +21,29 @@ import {
   MatEditClose
 } from './lens-directives';
 
-const EXPORTED_DECLARATIONS = [
-  MatPopoverEdit,
-  MatPopoverEditTabOut,
-  MatRowHoverContent,
-  MatEditLens,
-  MatEditRevert,
-  MatEditClose,
-  MatEditOpen
-];
-
 @NgModule({
   imports: [
     CdkPopoverEditModule,
     MatCommonModule,
   ],
   exports: [
-    ...EXPORTED_DECLARATIONS,
+    MatPopoverEdit,
```

Comment:
These changes are a workaround for an issue with ngcc.

Reply:
Is there a link/issue for this?

Reply:
There isn't, my changes in the PR for TS 4.4 on the framework side probably didn't cover the new generated code array spreads correctly. I'm looking into it now.

Reply:
PR to fix the compiler: https://github.com/angular/angular/pull/43618.